File: Java/average.java
package Java;
import java.util.Scanner;

public class average
{
    public static void main(String[] args)
{
    int num1, num2;
    double total;
    Scanner sc = new Scanner(System.in);
    System.out.print("Enter 1st number: ");
    num1 = sc.nextInt();
    System.out.print("Enter 2nd number: ");
    num2 = sc.nextInt();

    total = (num1+num2)/2;
    System.out.println("Average is: "+ (float)total);
}
}

File: C Programs/average.c
#include<stdio.h>

int main()
{
    int num1, num2;
    float total;
    printf("Enter 1st number: ");
    scanf("%d", &num1);
    printf("Enter 2nd number: ");
    scanf("%d", &num2);

    printf("%d %d \n", num1, num2);
    total = (float)(num1+num2)/2;
    printf("The average is: %.2f", total);

    return 0;
}